src/components/AppBar/DonateButton.style.ts

```diff
@@ -4,30 +4,24 @@ import { mobileBreakpoint } from 'assets/theme/sizes';
 import LinkButton from 'components/LinkButton';
 
 export const StyledDonateButton = styled(LinkButton).attrs(props => ({
-  variant: 'contained',
+  variant: 'outlined',
 }))`
+  background-color: white;
   margin: auto 0;
   display: inline-flex;
+  border: 2px solid ${COLOR_MAP.GRAY.LIGHT};
+  color: ${COLOR_MAP.PURPLE};
   box-shadow: none;
-  background-color: ${COLOR_MAP.PURPLE};
-  color: white;
-
   &:hover {
-    background-color: ${COLOR_MAP.PURPLE};
-    color: white;
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.12);
```

Comment:
i think we also want `background-color:white` here (it defaults to a gray background on hover)